statuspage/src/components/TableRow.vue

```diff
@@ -0,0 +1,82 @@
+<!-- This component encapsulates the table rows that is displayed -->
+<!-- when clicking on canary                               -->
+<template>
```

Comment:
Appreciate this, thanks. Nice work on the Vue.

Reply:
Thanks <3